h/sql_tasks/sql_file.py

```diff
@@ -0,0 +1,67 @@
+import os
+import os.path
+from dataclasses import dataclass
+from typing import List
+
+import jinja2
+
+from h.sql_tasks.sql_query import SQLQuery
+
+
+@dataclass
+class SQLFile:
+    """A class representing an SQL file with multiple queries."""
+
+    path: str
+    """Full path of the script."""
+
+    environment: dict
```

Comment:
I'd call this `jina_enviroment` or `template_enviroment`, otherwise from the caller looks like you should pass in os.env variables.

We might also just have `db_user` here and build the jinja environment here based on that.

Or, go full dataclasses and have a `class Enviroment`:  only with db_user for now`